masonry/src/widgets/flex.rs

```diff
@@ -1661,6 +1667,27 @@ mod tests {
         assert!(image_1 == image_2);
     }
 
+    #[test]
+    fn remove_child_after_expanded_child() {
+        let widget = Flex::column()
+            .with_flex_child(
+                SizedBox::new(Label::new("child 1"))
+                    .expand()
+                    .background(palette::css::PURPLE),
+                1.,
+            )
+            .with_flex_child(Label::new("child 2"), 1.0);
+
+        let window_size = Size::new(200.0, 150.0);
+        let mut harness = TestHarness::create_with_size(widget, window_size);
+
+        harness.edit_root_widget(|mut flex| {
+            let mut flex = flex.downcast::<Flex>();
+            Flex::remove_child(&mut flex, 1);
+        });
+        assert_render_snapshot!(harness, "flex_remove_child_after_expanded_child");
+    }
```

Comment:
Would you mind adding a similar test where the removed item is a spacer? I believe that this PR should also fix that, but I'd like to be sure.